drivers/node/ssh/ssh.go

```diff
@@ -55,6 +73,66 @@ func getKeyFile(keypath string) (ssh_pkg.Signer, error) {
 }
 
 func (s *ssh) Init() error {
+
+	nodes := node.GetWorkerNodes()
+	var err error
+
+	switch nodes[0].PlatformType {
```

Comment:
actually let's remove references to IKS itself.  Do this.

If torpedo is invoked with an empty ssh key and ssh password, then automatically install the ssh daemonset and configure the node driver to use that for all ssh operations.

End user should not needed to anything else.